http-svr/xmlrpc_client.ml

```diff
@@ -21,6 +21,8 @@ open D
 
 module E = Debug.Make(struct let name = "mscgen" end)
 
+let initialize_logging () = Debug.disable ~level:Syslog.Debug "mscgen"
```

Comment:
Does it work if we do it once here:
```ocaml
let () = Debug.disable ~level:Syslog.Debug "mscgen"
```
That would be more efficient than calling this in every rpc call.

Reply:
Yes, good point. I will change and test. thanks.